css/style.css

```diff
@@ -2489,7 +2489,7 @@ footer .grid--4 {
   }
 
   #post-results.grid-view {
-    grid-template-columns: 1fr;
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
I doubt there is a grid with two columns on mobile.

Reply:
Sure, I understand, but I have to adjust to the established design
<img width="213" alt="Captura de pantalla 2024-06-18 a las 12 26 24" src="https://github.com/mongabay/Mongabay-News-Wordpress-Theme/assets/157106140/d5fcb98b-e05c-43a8-8dbe-9c6d601496a6">

Reply:
I believe div with ID post-results is used for either search or taxonomy archive pages. If it's a different place then it should be different ID or class. Your change is affecting those places.

Reply:
Let's ommit post-results ID there and in template file switch to container class with grid--4 and gap--20 instead.